osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -189,9 +189,13 @@ private void start()
         }
 
         private ScheduledDelegate filterTask;
+        private string previousFilter = null;
 
         private void filterChanged()
         {
+            if (previousFilter == filter.Search)

```

Comment:
No, but I forgot to do the other thing I meant to do that justified that decision.